.github/workflows/changed_file.py

```diff
@@ -0,0 +1,21 @@
+import argparse
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--fileNameList', type=str, default='using_default_location')
```

Comment:
Why is the default `using_default_location`? It seems the default value is not used at all.

Reply:
Github PR conversation does not show the lack of EOF. You can see that this file does not have a EOF character if you go to the `Files Changed` tab of this PR.